docs/docs/configuration/settings.md

```diff
@@ -8,162 +8,171 @@ Onivim's settings are designed to be mostly compatible with [VSCode's User Setti
 
-- `editor.rulers` __(_list of int_ default: `[]`)__ - Render vertical rulers at given columns.
+-   `editor.scrollShadow` **(_bool_ default: `true`)** - When `true`, show a drop-shadow effect at the borders when there is additional content past the visible area.
 
-- `explorer.autoReveal` __(_string|bool_ default: `true`)__  - When `true`, the file explorer will jump to highlight the file current focused. When `false` the file explorer will remain static. If a string is entered it must be `"focusNoScroll"` which will still highlight the currently focused file in the file explorer but the file explorer will not scroll to it. Any other string supplied will be treated as if `false` was entered and the file explorer will remain static and not highlight the currently focused file.
+-   `editor.smoothScroll` **(_bool_ default: `true`)** - When `true`, smoothly scroll the editor when the viewport is adjusted due to a cursor motion.
 
-- `editor.scrollShadow` __(_bool_ default: `true`)__ - When `true`, show a drop-shadow effect at the borders when there is additional content past the visible area.
+-   `editor.zenMode.singleFile` **(_bool_ default: `true`)** - When `true`, the Onivim will automatically enter zen mode when started up with a single file. Zen mode hides most of the UI until disabled via the command pallette.
 
-- `editor.smoothScroll` __(_bool_ default: `true`)__ - When `true`, smoothly scroll the editor when the viewport is adjusted due to a cursor motion.
+-   `editor.zenMode.hideTabs` **(_bool_ default: `true`)** - When `true`, the Onivim will hide the buffer tabs from the user whilst in zen mode. Zen mode can be toggled in the command pallette, or automatically enabled with the `editor.zenMode.singleFile` configuration option.
 
-- `editor.zenMode.singleFile` __(_bool_ default: `true`)__ - When `true`, the Onivim will automatically enter zen mode when started up with a single file. Zen mode hides most of the UI until disabled via the command pallette.
+-   `files.exclude` **(_list of string_ default: `[".git", "_esy", "node_modules"]`)** - When using `Quick Open` or `Find in files`, Onivim will ignore the files inside the directories listed here
 
-- `editor.zenMode.hideTabs` __(_bool_ default: `true`)__ - When `true`, the Onivim will hide the buffer tabs from the user whilst in zen mode. Zen mode can be toggled in the command pallette, or automatically enabled with the `editor.zenMode.singleFile` configuration option.
+-   `search.exclude` **(_list of string_ default: `[]`)** - When using `Find in files` Onivim will not look at files located at the directories listed here, this inherit all the values from `files.exclude`
```

Comment:
Thanks for adding the docs here!

Are the extra changes just formatting changes? Would be helpful to minimize the changes just to the settings being added.

Reply:
Hi, i didnt notice this, it was the editor format autosave, but i will revert this and add just the necessary